src/main/kotlin/com/example/routes/tasks/taskRouter.kt

```diff
@@ -137,7 +137,8 @@ fun Route.tasks(context: DSLContext) {
                         var count = 0
                         for (question in task.details.questions) {
                             val givenAnswer = answers[question.number]
-                            if (givenAnswer == question.correctAnswerKey) count++
+                            if (givenAnswer == question.correctAnswerKey)
+                                count += question.answerWeightKey.toInt()
                         }
                         val score = (count.toDouble()/task.details.questions.size)*100
```

Comment:
Can't just divide by the number of questions in the list, the weight of the questions should also be taken into account

Reply:
Fixed with [c2e3db6](https://github.com/we-huddle/server/pull/43/commits/c2e3db6ee51cca4058328af208f9ed8b5a45a00e) 
cc: @Gravewalker666